apps/dashboard/src/@/actions/billing.ts

```diff
@@ -63,6 +70,13 @@ export type BillingPortalOptions = {
   teamSlug: string | undefined;
   redirectUrl: string;
 };
+
+export type BillingBillingPortalAction = (
```

Comment:
isn't this just `typeof redirectToBillingPortal`?

Reply:
would prefer to avoid duplicating it in that case, fine to export the type separately but just do the `typeof`

Reply:
updated